_includes/manuals/1.12/5.8.2_encryption_rb.md

```diff
@@ -0,0 +1,2 @@
+
+As described in [5.2.10](/manuals/{{page.version}}/index.html#5.2.10PasswordEncryption), passwords stored locally in Foreman's DB are encrypted. In order for multiple Foreman instances to encrypt and decrypt passwords correctly, they all need to have the same encryption key. 
```

Comment:
I'd suggest adding the file path here, e.g. "[..] have the same encryption key defined in `/etc/foreman/encryption_key.rb`."